src/components/Queue.vue

```diff
@@ -70,23 +66,20 @@ export default {
             validate: false,
             fail: false,
             message: '',
-            clearMessage: true
         }
     },
     watch: {
-      queue: {
-          handler: function(val, oldVal) {
-              let queueState = val.sendingStatus || false;
-              if(!queueState && this.netMgr.online) {
-                  this.showValidate();
-              } else if(!queueState) {
-                  this.spinner = false;
-              }
-
-          },
-          deep: true
-      },
-
+        queue: {
+            handler: function(val) {
+                let queueState = val.sendingStatus || false;
```

Comment:
This was added in another PR. But seeing as we're making most things ES5 I guess we should change it to a var. Babel will convert it to a var either way.